platform/viewer/src/appExtensions/MeasurementsPanel/ConnectedMeasurementTable.js

```diff
@@ -142,8 +162,30 @@ function convertTimepointsToTableData(timepoints) {
   ];
 }
 
+function updateWarnings(measurementCollection) {
+  measurementCollection;
+}
```

Comment:
I'm not sure I understand what this function is doing

Reply:
Probably refactor leftover.  
Removed it.